e2e/app/element/react-select.ts

```diff
@@ -24,9 +25,10 @@ export default class ReactSelect extends BaseElement {
     await option.click({ delay: 20 });
   }
 
-  async waitForOption(optionText: string): Promise<ElementHandle> {
+  async waitForOption(optionText: string): Promise<Link> {
     const selector = this.getOptionXpath(optionText);
-    return this.page.waitForXPath(selector, { visible: true });
+    await this.page.waitForXPath(selector, { visible: true });
+    return new Link(this.page, selector);
```

Comment:
Return `link` instead `ElementHandle` because `.click` function checks and waits until the element stops moving position.